lncfg/dev.go

```diff
@@ -24,6 +24,12 @@ func (d *DevConfig) ChannelReadyWait() time.Duration {
 	return 0
 }
 
+// GetUnsafeDisconnect returns the config value, which is always false for
+// production build.
+func (d *DevConfig) GetUnsafeDisconnect() bool {
+	return false
```

Comment:
Shouldn't this be `true` to not change the default behavior of `lnd`?

Reply:
I think by default we should not allow unsafe disconnection. IIUC, that's the reason the flag was deprecated?

Reply:
but since some node runners might be depending on the current behaviour (disconnect can be used to re-set all connections with a peer), should we keep this `true` for now until we have the channel-level disconnect RPC in place?